10-get-the-photo/SOLUTION.md

```diff
@@ -1,3 +1,17 @@
 # Solution
 
-TBD
+## Other solutions - [mwhitman189](https://github.com/mwhitman189)
+This seems like a problem screaming for '.reduce', so here it is:
+
+```js
+function getPhotos(arrObject) {
+  const photos = arrObject.reduce((acc, curr) => {
```

Comment:
@mwhitman189 FYI `arrObject` is kind of like our inside joke 😆 .. so it is better to use input names that reflect the actual values `photos` for example